addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -357,15 +358,16 @@ function registerThemeHomepageTour(name, steps) {
     }
     return registerWebsitePreviewTour(name, {
         url: '/',
-        edition: true,
-        sequence: 1010,
+        sequence: 50,
```

Comment:
Why do we need to change the sequence?

Reply:
Otherwise other tours take the priority (e.g. the blogs one). It was not a problem when we arrived in edit mode, but it is now that we arrive on the main homepage with the backend header, with several tours competing.